### PR TITLE
Updates for Google Shopping ads Channel v3.0

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Release Notes and install guide for Google Shopping Channel for v3.0, which includes the new **Enable Log Service** option. Updated the screen shot and added a note about enabling logging service.

## Affected DevDocs pages

https://devdocs.magento.com/extensions/google-shopping-ads/
https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/

whatsnew
Updated the [Google Shopping ads Install documentation](https://devdocs.magento.com/extensions/google-shopping-ads/) and [Release Notes](https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/) for version 3.0.

### CODE REVIEW

File: extensions/google-shopping-ads/index.md [hunk 1]
```diff
@@ -61,10 +61,14 @@ To add the Google API key:
 
     ![Add the API Key](images/config-api-key.png)
 
+    {:.bs-callout .bs-callout-info}
+You can also [Enable Service Logs](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) during this step. This option is disabled by default and should only be used when needed for troubleshooting, as continued logging will negatively impact performance.
```
Comment (lorikrell) [inline extensions/google-shopping-ads/index.md]: Links does not work. Not in production: https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html
Also would check if you need to indent the content for the note to properly align under the indented note code. If it should not indent, remove indent from note code.

Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: Also if this is strictly for version 2.0.0, may need to note that.

**Comment by lorikrell** [comment]
Also update the whatsnew with a link to the devdocs update file. Like: 
`Updated [Google Shopping ads Install](link) for version 2.0 to include the Enable Service Logs option.`

File: extensions/google-shopping-ads/index.md (hunk 1 above)
Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: Removed the first link comment as the merchdocs 2.0 PR is not merged yet. 👍

Comment (lorikrell) [inline extensions/google-shopping-ads/index.md]: Suggestion: if this is strictly for version 2.0.0, may need to note that.

Comment (hickskenh) [reply extensions/google-shopping-ads/index.md]: added "Beginning with version 2.0.0, you...."

**Comment by hickskenh** [comment]
running tests

File: extensions/google-shopping-ads/index.md (hunk 1 above)
Comment (hickskenh) [reply extensions/google-shopping-ads/index.md]: this has been updated to 3.0 now.

File: extensions/google-shopping-ads/index.md [hunk 2]
```diff
@@ -61,10 +61,14 @@ To add the Google API key:
 
     ![Add the API Key](images/config-api-key.png)
 
+    {:.bs-callout .bs-callout-info}
```
Comment (jeff-matthews) [inline extensions/google-shopping-ads/index.md]:
```suggestion
    {:.bs-callout-info}
```

Comment (jeff-matthews) [reply extensions/google-shopping-ads/index.md]: Only one class name is required now.

File: extensions/google-shopping-ads/release-notes/index.md [hunk 3]
```diff
@@ -17,6 +17,29 @@ The release notes include:
 -   {:.fix}Fixes and improvements
 -   {:.bug}Known issues
 
+### v3.0.0
+
+Google Shopping ads Channel 3.0.0 is generally available for versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.new} **Magento Log Exceptions**: <!--CHAN-3439-->Implemented **Enable Services Logs** feature to help in troubleshooting. See the [Channels](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) topic in the Configuration Reference.
```
Comment (jeff-matthews) [inline extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
- {:.new} **Magento Log Exceptions**: <!--CHAN-3439-->Implemented **Enable Services Logs** feature to help with troubleshooting. See the [Channels](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) topic in the Configuration Reference.
```

File: extensions/google-shopping-ads/index.md (hunk 2 above)
Comment (hickskenh) [reply extensions/google-shopping-ads/index.md]: committed suggestion

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**